diff -Naur xia-1.00/dom-core-append_node.ads xia-1.00.new/dom-core-append_node.ads
--- xia-1.00/dom-core-append_node.ads	2005-03-05 14:46:57.000000000 +0100
+++ xia-1.00.new/dom-core-append_node.ads	2014-02-04 15:58:19.778372317 +0100
@@ -29,7 +29,7 @@
 ------------------------------------------------------------------------
 
 -- Append a node to an XMLAda DOM node list
-procedure Dom.Core.Append_Node
+procedure DOM.Core.Append_Node
   (List : in out Node_List;
    -- list to which to append the node
 
diff -Naur xia-1.00/mckae-xml-xpath-query_handling.adb xia-1.00.new/mckae-xml-xpath-query_handling.adb
--- xia-1.00/mckae-xml-xpath-query_handling.adb	2005-03-05 14:49:26.000000000 +0100
+++ xia-1.00.new/mckae-xml-xpath-query_handling.adb	2014-02-04 14:19:48.914458112 +0100
@@ -55,7 +55,7 @@
 
       Path := Locations.Get_Path;
 
-      Path.Absolute := Xpath_Query(1) = '/';
+      Path.Absolute := Xpath_Query(Xpath_Query'First) = '/';
       Path.Path(Path.Steps).Output_Step := True;
 
       return Path;
diff -Naur xia-1.00/mckae-xml-xpath-xia.ads xia-1.00.new/mckae-xml-xpath-xia.ads
--- xia-1.00/mckae-xml-xpath-xia.ads	2005-03-05 14:49:26.000000000 +0100
+++ xia-1.00.new/mckae-xml-xpath-xia.ads	2014-02-04 14:19:48.914458112 +0100
@@ -39,7 +39,7 @@
 
 with Dom.Core;
 
-package Mckae.XML.XPath.XIA is
+package McKae.XML.XPath.XIA is
 
    function XPath_Query
      (N     : Dom.Core.Node;
