Index: cpu/init.cc
===================================================================
--- cpu/init.cc	(revision 11662)
+++ cpu/init.cc	(working copy)
@@ -885,7 +885,7 @@
   BX_CPU_THIS_PTR vmcshostptr = 0;
   /* enable VMX, should be done in BIOS instead */
   BX_CPU_THIS_PTR msr.ia32_feature_ctrl =
-    /*BX_IA32_FEATURE_CONTROL_LOCK_BIT | */BX_IA32_FEATURE_CONTROL_VMX_ENABLE_BIT;
+    BX_IA32_FEATURE_CONTROL_LOCK_BIT | BX_IA32_FEATURE_CONTROL_VMX_ENABLE_BIT;
 #endif
 
 #if BX_SUPPORT_SVM
