diff -r e0a1f2e18944 Makefile
--- a/Makefile	Tue Jun 01 20:49:59 2010 +0300
+++ b/Makefile	Thu Jul 22 10:09:19 2010 +0300
@@ -17,6 +17,7 @@
 PREFIX?=$(HOME)/libraries/ahven
 INSTALL=install
 OS_VERSION?=unix
+LIBDIR?=$(PREFIX)/lib
 
 SOURCES=src/ahven-framework.adb src/ahven-framework.ads \
 	src/ahven-listeners-basic.adb src/ahven-listeners-basic.ads \
@@ -89,11 +90,11 @@
 
 install_lib:
 	mkdir -p $(PREFIX)/include/ahven
-	mkdir -p $(PREFIX)/lib/ahven
+	mkdir -p $(LIBDIR)/ahven
 	mkdir -p $(PREFIX)/lib/gnat
 	$(INSTALL) -m 644 $(SOURCES) $(PREFIX)/include/ahven
-	$(INSTALL) -m 444 $(ALI_FILES) $(PREFIX)/lib/ahven
-	$(INSTALL) -m 644 lib/$(STATIC_LIBRARY) $(PREFIX)/lib/ahven
+	$(INSTALL) -m 444 $(ALI_FILES) $(LIBDIR)/ahven
+	$(INSTALL) -m 644 lib/$(STATIC_LIBRARY) $(LIBDIR)/ahven
 	$(INSTALL) -m 644 $(GPR_FILE) $(PREFIX)/lib/gnat
 
 check: build_tests
